<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>مكتبة الراشدين</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_loginName">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>arial</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string extracomment="يرجى ادخال اسم المستخدم في الخانة المخصصة له"/>
       </property>
       <property name="whatsThis">
        <string extracomment="يرجى ادخال اسم المستخدم في الخانة المخصصة"/>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{

 margin-bottom: 2px;   
    padding: 7px 8px;
        padding-top: 7px;
        padding-bottom: 7px;
    outline: medium none;
        outline-width: medium;
        outline-style: none;
        outline-color: -moz-use-text-color;
    color: #333;
    background-color: #FFF;
    background-repeat: no-repeat;
    background-position: right center;
    border: 1px solid #c5c5c5;
        border-top-width: 1px;
        border-bottom-width: 1px;
        border-top-style: solid;
        border-bottom-style: solid;
        border-top-color: #c5c5c5;
        border-bottom-color: #c5c5c5;      
    border-radius: 3px;
        border-top-left-radius: 3px;
        border-top-right-radius: 3px;
        border-bottom-right-radius: 3px;
        border-bottom-left-radius: 3px;
    vertical-align: middle;
}
QLineEdit:hover
{
	 border: 1px solid #00bbff;
	 border-top-color: #00bbff;
        border-bottom-color: #00bbff;  

}</string>
       </property>
       <property name="text">
        <string>admin</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>إسم المستخدم</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_passwordLogin">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>arial</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{

 margin-bottom: 2px;   
    padding: 7px 8px;
        padding-top: 7px;
        padding-bottom: 7px;
    outline: medium none;
        outline-width: medium;
        outline-style: none;
        outline-color: -moz-use-text-color;
    color: #333;
    background-color: #FFF;
    background-repeat: no-repeat;
    background-position: right center;
    border: 1px solid #c5c5c5;
        border-top-width: 1px;
        border-bottom-width: 1px;
        border-top-style: solid;
        border-bottom-style: solid;
        border-top-color: #c5c5c5;
        border-bottom-color: #c5c5c5;      
    border-radius: 3px;
        border-top-left-radius: 3px;
        border-top-right-radius: 3px;
        border-bottom-right-radius: 3px;
        border-bottom-left-radius: 3px;
    vertical-align: middle;
}
QLineEdit:hover
{
	 border: 1px solid #00bbff;
	 border-top-color: #00bbff;
        border-bottom-color: #00bbff;  

}</string>
       </property>
       <property name="text">
        <string>admin</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>كلمة المرور</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BT_login">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
 position: relative;
    padding: 7px 12px;
        padding-top: 7px;
        padding-bottom: 7px;
    font-size: 15px;
    font-family: Ubuntu;
    color: #ffffff;
    white-space: nowrap;
    background-color: #E36C43;
   
    border-radius: 3px;
        border-top-left-radius: 3px;
        border-top-right-radius: 3px;
        border-bottom-right-radius: 3px;
        border-bottom-left-radius: 3px;
    border: 1px solid #E36C43;
        border-top-width: 1px;
        border-bottom-width: 1px;
        border-top-style: solid;
        border-bottom-style: solid;
        border-top-color: #E36C43;
        border-bottom-color: #E36C43;
    vertical-align: middle;
}

QPushButton:hover
{
background-color: #DD4814;
color: #ffffff;

}
</string>
       </property>
       <property name="text">
        <string>دخول</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BT_cancel">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
 position: relative;
    padding: 7px 12px;
        padding-top: 7px;
        padding-bottom: 7px;
    font-size: 15px;
    font-family: Ubuntu;
    color: #ffffff;
    white-space: nowrap;
    background-color: #E36C43;
   
    border-radius: 3px;
        border-top-left-radius: 3px;
        border-top-right-radius: 3px;
        border-bottom-right-radius: 3px;
        border-bottom-left-radius: 3px;
    border: 1px solid #E36C43;
        border-top-width: 1px;
        border-bottom-width: 1px;
        border-top-style: solid;
        border-bottom-style: solid;
        border-top-color: #E36C43;
        border-bottom-color: #E36C43;
    vertical-align: middle;
}

QPushButton:hover
{
background-color: #DD4814;
color: #ffffff;

}
</string>
       </property>
       <property name="text">
        <string>الغاء</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="LB_msg">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
